### PR TITLE
feat: add evm test data scripts

### PR DESCRIPTION
**Pull request by sambukowski**
Adding Forge scripts for automating evm test data.

Resolves https://github.com/astriaorg/dev-cluster/issues/21

### CODE REVIEW

File: packages/evm-test-data/.gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+/broadcast/
```
Comment (steezeburger) [inline packages/evm-test-data/.gitignore:6]:
```suggestion
!/broadcast
```

File: packages/evm-test-data/.gitignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+/broadcast/*/31337/
```
Comment (steezeburger) [inline packages/evm-test-data/.gitignore:7]:
```suggestion
/broadcast/*/912559/
```
This number is the chain id. Using these patterns will ensure we keep the `broadcast` directory, but not commit the contents.

Comment (sambukowski) [reply packages/evm-test-data/.gitignore:7]: fixed this

File: packages/evm-test-data/.gitignore (hunk 1 above)
Comment (sambukowski) [reply packages/evm-test-data/.gitignore]: fixed this

File: README.md [hunk 3]
```diff
@@ -24,6 +25,14 @@ dotenv -- bash -c 'RUST_LOG=debug forge create \
   --rpc-url $JSON_RPC \
   src/Weth9.sol:WETH9'
 
+# deploy test token
```
Comment (steezeburger) [inline README.md:28]: We can delete this since we removed the SteezeToken :( haha

**Review by steezeburger — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sambukowski) [reply README.md]: removed! rip steezetoken

**Review by steezeburger — APPROVED** [review]

**Pull request merged**